lib/monitoring/aws-loadbalancing/LoadBalancerMetricFactory.ts

```diff
@@ -12,29 +12,27 @@ import { MetricFactory, MetricWithAlarmSupport } from "../../common";
 // It's not possible to use instanceOf with typescript interfaces, so we have to use type guards to differentiate
 // between the interfaces. As another problem, the LoadBalancer/TargetGroup for both Network and Application types
 // don't have distinguished fields that could be used to differentiate between both types, so we resort to using
-// the name of the class below.
-//
-// Ideally the 2 interfaces would provide a specific field to distinguish both types.
+// checking for unique methods in their metrics interfaces.
```

Comment:
Checking on unique metrics methods still feels iffy but I guess that's all we have right now.